lib/repl.js

```diff
@@ -1256,13 +1256,16 @@ function defineDefaultCommands(repl) {
       try {
         var stats = fs.statSync(file);
         if (stats && stats.isFile()) {
+          this.editorMode = true;
+          REPLServer.super_.prototype.setPrompt.call(this, '');
           var data = fs.readFileSync(file, 'utf8');
           var lines = data.split('\n');
-          this.displayPrompt();
           for (var n = 0; n < lines.length; n++) {
             if (lines[n])
               this.write(`${lines[n]}\n`);
           }
+          this.turnOffEditorMode();
+          this.write('\n');
```

Comment:
Does turning the `turnOffEditorMode` and `write("\n")` around also change the order of the blank line and the `undefined` (looking at the test)? If so, I think that would be nice but it is not a blocker at all.

Reply:
@BridgeAR rather than hold up this PR, I will do this separately.